src/App.js

```diff
@@ -0,0 +1,91 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+
+<nav>
+        
+            <a className='logo' href="/home.html"> LOGo</a>   
+      
+
+        <ul className='links'>
```

Comment:
use either double quotes either single quotes in the markup, code should be consistent. Do not use different quotes in the similar places of code.